### ISSUE TITLE
Increase visibility of "Leave the line" focus indicator

### ISSUE THREAD

**Issue by teshsopher** [tl-its-umich-edu/remote-office-hours-queue#158]
"Leave the line" focus treatment is very subtle: 

<img width="196" alt="image" src="https://user-images.githubusercontent.com/63423382/92797684-4c519580-f380-11ea-8632-564e146fe698.png">

The outline/highlight should be more visible and have sufficient contrast

**Comment by jlost** [comment]
Looks like only Join Meeting uses this particular style now.  
@teshsopher  How does this look?:
![image](https://user-images.githubusercontent.com/11338926/97884849-9443c780-1cf4-11eb-9db5-ed22ecc2abc3.png)

**Comment by jlost** [comment]
I could also change the style to "primary" (blue) if that's preferred.

**Comment by teshsopher** [comment]
Tested and working in QA
